brion/plugin/spikeReportASCII.cpp

```diff
@@ -192,7 +192,7 @@ void _parse( Spikes& spikes, const std::string& filename,
                                         filename + " failed at line " +
                                         std::to_string( lineNumber )));
        }
-       spikes.push_back( spike );
+       spikes.emplace_back( spike );
```

Comment:
Does it make a difference with a copyable object that you're not moving (although in this case even moving and copying are the same).